sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.dto.ts

```diff
@@ -120,3 +122,14 @@ export class CreateApplicationOfferingChangeRequestAPIInDTO {
   @MaxLength(REASON_MAX_LENGTH)
   reason: string;
 }
+
+/**
+ * Details to update the application offering change request by student.
+ */
+export class UpdateApplicationOfferingChangeRequestAPIInDTO {
+  @IsOptional()
+  @IsBoolean()
+  studentConsent?: boolean;
```

Comment:
Why is student consent an optional field?

Reply:
@dheepak-aot I kept `studentConsent` an optional because I am not making use of `studentConsent` while declining a application offering change request.

Reply:
fixed it.